diff --git a/Cargo.toml b/Cargo.toml
index b8848f3..94457ec 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -35,6 +35,7 @@ autocfg = "1"
 [dependencies]
 serde = { version = "1.0", optional = true, default-features = false }
 rayon = { version = "1.4.1", optional = true }
+proc-macro2 = { version = "=1.0.42" }
 
 # Internal feature, only used when building as part of rustc,
 # not part of the stable interface of this crate.
