diff --git a/Cargo.toml b/Cargo.toml
index 0a5851f..a7367ac 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -34,6 +34,8 @@ rocket = { default-features = false, optional = true, version = "0.5.0-rc.1" }
 serde = { default-features = false, optional = true, version = "1.0" }
 serde_json = { default-features = false, optional = true, version = "1.0" }
 tokio-postgres = { default-features = false, optional = true, version = "0.7" }
+csv = "=1.1"
+proc-macro2 = { version = "=1.0.42" }
 
 [dev-dependencies]
 bincode = "1.3"
